libflux/src/core/semantic/nodes.rs

```diff
@@ -70,6 +71,7 @@ impl fmt::Display for Error {
                 write!(f, "error {}: invalid import path {}", loc, path)
             }
             Error::InvalidReturn(loc) => write!(f, "error {}: return not valid in file block", loc),
+            Error::ASTError(s, loc) => write!(f, "error {}: {}", s, loc),
```

Comment:
Should this `write!(f, "error {}: {}", s, loc)` be `write!(f, "error {}: {}", loc, s)`?

Reply:
Yes. This was a mistake.